lbz/authz.py

```diff
@@ -76,6 +76,10 @@ def _set_policy(self, auth_jwt: str, policy_override: dict = None):
         elif issuer != ALLOWED_ISS:
             raise PermissionDenied(f"{issuer} is not an allowed token issuer")
 
+    def _raise_error(self):
```

Comment:
You can be more direct and name it `_raise_permission_denied()` 😉  It will simplify reading the code in the future as you won't have to check implementation to be sure what will be raised.